repository/postgresql.go

```diff
@@ -0,0 +1,32 @@
+package repository
+
+import (
+	"context"
+	"fmt"
+	"github.com/jackc/pgx/v5"
+)
+
+type StorageConfig struct {
+	username, password, host, port, database string
+}
+
+func NewClient() {
+	var sc = StorageConfig{
+		"admin",
```

Comment:
Please use env vars instead of a hardcoded values

Reply:
https://gobyexample.com/environment-variables

Reply:
Also you can use .env file for storing them https://towardsdatascience.com/use-environment-variable-in-your-next-golang-project-39e17c3aaa66?gi=711dd940ab79

Reply:
https://en.wikipedia.org/wiki/Environment_variable